### PR TITLE
Correct return null should always be last defined

### PR DESCRIPTION
**Pull request by JohJohan**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

### CODE REVIEW

**Comment by vincentchalamon** [comment]
Hello @JohJohan,
Is there any reason/PSR for this change?
Thanks

**Comment by JohJohan** [comment]
Hello @vincentchalamon,

I had a look at your code and saw you had a 'null|mixed' i think its better to return null as last to make it more clear :)

**Comment by vincentchalamon** [comment]
But is there a PSR or any recommandation about it, from the PHP (or Symfony) community? Or is it your personal recommandation?

**Comment by JohJohan** [comment]
You can see that Symfony also implements this here https://github.com/symfony/symfony/blob/4d757b5382e1356c96cf666392b301708bf97c38/src/Symfony/Component/HttpFoundation/Cookie.php#L191

**Comment by JohJohan** [comment]
Okay will do didn't realise phpcs fixer was part of the project. Also why isnt `.php_cs.cache` in the gitignore file? And you could include phpcs fixer via composer

**Comment by vincentchalamon** [comment]
Would be a great improve, feel free to add it in your PR :wink:

**Comment by vincentchalamon** [comment]
I'm about to remove PHP 5.5 support from travis as this PHP version is not supported anymore (cf. http://php.net/supported-versions.php)

**Comment by JohJohan** [comment]
Okay i think its all done, let me know if it is okay! I also did a rebase.

**Comment by JohJohan** [comment]
Did you update readme i still see support for php 5.5+

**Comment by vincentchalamon** [comment]
I didn't change the project dependencies about PHP 5.5 because it would be a BC break. I just removed the travis analysis about it. The PHP 5.5 dependency will be removed in a later PR

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ If you happen to find a bug, we kindly request you to report it using GitHub by
 ### Matching coding standards
 
 The CoopTilleulsForgotPasswordBundle project follows [Symfony coding standards](https://symfony.com/doc/current/contributing/code/standards.html).
-But don't worry, you can fix CS issues automatically using the [PHP CS Fixer](http://cs.sensiolabs.org/) tool
+But don't worry, you can fix CS issues automatically using the [PHP CS Fixer](http://cs.sensiolabs.org/) tool. Run phpcs fixer:
 
 ```bash
-php-cs-fixer.phar fix
+vendor/bin/php-cs-fixer fix
```
Comment (vincentchalamon) [inline CONTRIBUTING.md]: Please also update `.travis.yml` configuration file

**Comment by JohJohan** [comment]
Why is travis failing on php 7.1 and php 7.2?

**Comment by vincentchalamon** [comment]
Seems some travis cache issues maybe, I'm checking it

**Comment by vincentchalamon** [comment]
Seems travis doesn't like a long `composer update` on dependencies low. Could you try to add a `travis_wait` before this command?
```yaml
# .travis.yml
... travis_wait composer update ...
```
(sources: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received)

**Comment by vincentchalamon** [comment]
Something might be wrong in your PR, the analysis is working well on master. Maybe any issue about composer cache or something else.

**Comment by vincentchalamon** [comment]
Well done @JohJohan, thank you very much :-)

**Pull request merged**